test/rename_1p_crates/BUILD.bazel

```diff
@@ -0,0 +1,60 @@
+load("//rust:defs.bzl", "rust_library", "rust_test")
+
+rust_library(
```

Comment:
The ideal would be if the import! macro could deal with both states of `rename_1p_crates`, when the flag is disabled it would just assume crate names as `rules_rust` do today. With that we could have a passing integration test both ways.

But the macro is not yet upstreamed, so I think it's fine to delete these tests for the time being. What do you think?

Reply:
The macro I've been prototyping does take a `bool` argument that disables the renaming, so that it is effectively a no-op; that way all the relevant files in the repo can be migrated, then the macro definition can be modified so that it starts renaming everything all at once.

I think that could help in the integration test, though I'm not sure how to (implicitly) plumb that bool from the .bazelrc file into the macro invocation. Would that mean the macro has to read the .bazelrc file at compilation time?

Also - I'm happy to start upstreaming the macro prototype as well, if there's a way to depend on external crates here (aho_corasick, quickcheck, etc).

Reply:
I think we can talk about it in the PR that adds the macro :) By the way external crates are possible (we use cargo raze for this). The potential idea for plumbing the boolean is to use an environment variable and read its value from the macro.